lookout/style/format/postprocess.py

```diff
@@ -145,21 +170,21 @@ def filter_uast_breaking_preds(
             # to handle mixed indentations, we include the `VirtualNode` following the predicted
             # one in the output predicted string, and start the rest of the sequence one
             # `VirtualNode` further to avoid its repetitions
-            content_after = content_before[:vn_y.start.offset] \
-                + output_pred \
-                + content_before[vn_y.start.offset + len(vn_y.value)
-                                 + len(vnodes[cur_i + 1].value):]
+            content_after = (file_content[:vn_y.start.offset]
+                             + output_pred
+                             + file_content[vn_y.start.offset + len(vn_y.value)
```

Comment:
Let's move `vn_y.start.offset + len(vn_y.value) + len(vnodes[cur_i + 1].value)` to a new variable and use it here.

Reply:
I agree that it's not readable. I'm currently discussing this part of the function with Warren to see how I can further refactor it.